src/Libraries/RevitNodes/Elements/Group.cs

```diff
@@ -0,0 +1,191 @@
+﻿using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using Autodesk.Revit.DB;
+using Revit.GeometryConversion;
+using RevitServices.Transactions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Autodesk.Revit.Creation;
+
+namespace Revit.Elements
+{
+    public class Group : Element
+    {
+        #region Internal properties
+
+        /// <summary>
+        /// An internal handle on the Revit group
+        /// </summary>
+        internal Autodesk.Revit.DB.Group InternalGroup
+        {
+            get; private set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
+        /// </summary>
+        [SupressImportIntoVM]
+        public override Autodesk.Revit.DB.Element InternalElement
+        {
+            get { return InternalGroup; }
+        }
+
+        #endregion
+
+        #region Private constructors
+
+        /// <summary>
+        /// Private constructor
+        /// </summary>
+        private Group(Autodesk.Revit.DB.Group group)
+        {
+            SafeInit(() => InitGroup(group));
+        }
+
+        /// <summary>
+        /// Initialize a group element
+        /// </summary>
+        private void InitGroup(Autodesk.Revit.DB.Group group)
+        {
+            InternalGroup = group;
+            InternalElementId = group.Id;
+            InternalUniqueId = group.UniqueId;
+        }
+
+        #endregion
+
+        #region public static constructors
+
+        /// <summary>
+        /// Creates a new type of group, with all of the input elements.
+        /// </summary>
+        /// <param name="elements">Revit elements to group.</param>
+        /// <returns>The new group type</returns>
+        public static Group FromElements(List<Element> elements)
+        {
+            List<ElementId> elementIds = elements.Select(x => x.InternalElement.Id).ToList();
+            TransactionManager.Instance.EnsureInTransaction(Document);
+            var group = Document.Create.NewGroup(elementIds).ToDSType(true) as Group;
+            TransactionManager.Instance.TransactionTaskDone();
+            return group;
+        }
+
+        /// <summary>
+        /// Place an instance of a Model Group into the Autodesk Revit document, using a location and a group type.
+        /// </summary>
+        /// <param name="location">The point to place the group.</param>
+        /// <param name="groupType">The type of group to place.</param>
+        /// <returns>The new group instance.</returns>
+        public static Group PlaceInstance(Autodesk.DesignScript.Geometry.Point location, Element groupType)
```

Comment:
not 100% sure of node name, maybe `MakeInstanceAtPoint` ? `Place` could be ambiguous whether it places something existing or creates a new group. To be discussed with Sol/Minjie maybe ?
Also, we use `Location` instead of `Point` for query node, maybe we'd use `MakeInstanceAtLocation` ?